propagation/usecases.md

```diff
@@ -0,0 +1,167 @@
+discover how APEx can provide support.
+:::
+
+### Examples
+
+* The project has created an openEO UDP or OGC Application Package that calculates a new air quality index and wants to
+  share this service with the EO community.
+* The project has developed a new machine learning model encapsulated in an openEO UDP or Application Package and wants
+  to ensure it runs stably and within a controlled cost range.
+* The project has developed an OGC Application Package and is seeking support to host it on an APEx-compliant platform.
+
+### Solution
+
+The [APEx Algorithm Onboarding](onboarding.md) service offers comprehensive support for integrating your algorithm into
+the APEx Algorithm Catalogue, ensuring its availability for reuse within the EO community. The service includes:
+
+* Assistance in selecting an APEx-compliant algorithm hosting platform to guarantee that your algorithm is available for
+  execution.
+* Provision of tools for automated testing and benchmarking of your algorithm.
+* Support and resources for registering your algorithm in the APEx Algorithm Catalogue.
+
+### Outcome
+
+The algorithm is available in the APEx Algorithm Catalogue, enhancing its visibility and usability within
+the EO community as an APEx-compliant service.
+
+## Use Case 5: Automating EO Data Processing Pipelines for Large-Scale Data Processing
+
+### Scenarios
+
+* A project is currently testing and validating an openEO or OGC Application Package based data processing pipeline on a
+  limited subset of data and aims to upscale this pipeline for production and large-scale data processing.
+
+:::{.callout-tip}
+
+# Is your pipeline not yet implemented in openEO or packaged as an OGC Application Package?
+
+APEx provides a set of initial [guidelines](../interoperability/algohosting.md) and
+online [resources](index.qmd#how-to-build-an-on-demand-service) to help you get started. If your project requires
+additional assistance in making the data processing pipeline APEx-compliant, the [APEx Algorithm Porting](porting.md)
+service is available to provide further support.
+:::
+
+### Examples
+
+* The project has developed an openEO processing graph or OGC Application Package-based workflow to generate a new air
+  quality index. So far, the testing and validation of this workflow have been carried out over a limited geographical
+  area, and the project now aims to generate the index at the country level.
+* The project has discovered an algorithm in the APEx Algorithm Catalogue for generating a new air quality index and
+  aims to execute this algorithm on all historical data for the entire continent of Europe.
+
+### Solution
+
+The [APEx Upscaling Service](upscaling.md) offers tools and support to assist projects in automating and scaling their
+data processing pipelines.
+These tool may include:
+
+* **Project-Specific Workspace**
+  <br/>A dedicated workspace for managing and monitoring your processing pipelines.
```

Comment:
This is again a very broad and vague statement, I propose to be more concrete, like:
```suggestion
  <br/>A Jupyterlab environment allowing to start and monitor long-running jobs.
```